src/test/java/edu/oakland/test/database/DatabaseCommInterfaceTest.java

```diff
@@ -0,0 +1,119 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabaseCommInterface;
+import edu.oakland.production.database.DatabaseCommInterfaceClass;
+import edu.oakland.production.database.DatabasePersistentStorage;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database CommInterface Test")
+public class DatabaseCommInterfaceTest {
+
+  @Test
+  @DisplayName("rfid going in is Rfid coming Out")
+  void rfidInIsRfidOut() {
+    int i = 10;
+    DatabaseCommManagerStub dcm = new DatabaseCommManagerStub();
+    assertEquals(i, dcm.passGetRfidRequest());
+  }
+  
+  @Test
+  @DisplayName("Offset matches Datapoint")
+  void offsetMatchDataPoint() {
+    
+    DatabaseCommManagerStub dcm = new DatabaseCommManagerStub();
+
+    float lat = (float) (Math.random() * 360 - 180);
+    float lng = (float) (Math.random() * 360 - 180);
+    LocationDataPoint ldp = new LocationDataPoint(lat, lng,
+        LocalDateTime.of((int) (Math.random() * 50 + 1970),
+        (int) (Math.random() * 12 + 1),
+        (int) (Math.random() * 28 + 1), (int) (Math.random() * 24),
+        (int) (Math.random() * 60)));
+    ldp = null;    
```

Comment:
We shouldn't be setting what we just randomly generated to `null`. `null` is an edge case that should be handled separately, and should not be used as a default test case. Instead, let's pass our randomly generated `LocationDataPoint` to the the `DatabaseCommManagerStub`. You may need to add a setter to the stub to do this.